src/items/item.js

```diff
@@ -57,6 +60,19 @@ export class Item extends ClickableObject {
                 }
             });
         this.displayHeight = this.sprite.displayHeight;
+
+        // Add text in top left for stackable items
+        this.numItemsText = this.scene.add
+            .text(x - 15, y - 15, this.numItems, {
+                font: "10px runescape",
+                fill: "orange",
```

Comment:
Mentioned on league. 1-99,999 = yellow, 100k-9999k = white, 10m+ = green

Reply:
Do you want yellow or the orange that I'm using rn?

Reply:
I just tried the yellow and I think orange looks better unless you have a specific hex value of yellow you want. I implemented the white + k/green + m with 1 decimal place after each. Ex: 1.2k, 5.1m. Lmk if that's what you were thinking of